### ISSUE TITLE
ng add schematics has a directory assumption which breaks adding to a specific project

### ISSUE THREAD

**Issue by armodev** [ngrx/platform#3540]
### Minimal reproduction of the bug/regression with instructions

👀 Please provide instructions how to reproduce schematics behavior in Stackblitz.

### Minimal reproduction of the bug/regression with instructions

1. In an ng cli app created with `--create-application false`, create a library `ng g lib whatever-works` (you should end up having a directory `projects/whatever-works`
2. As per [docs](https://v8.ngrx.io/guide/store/install#optional-ng-add-flags) run schematics to add in a specific project: `ng add @ngrx/store --project whatever-works`
3. The schematics will fail with the following error:
**Specified module path /projects/whatever-works/src/lib/app does not exist**
Indee, `app` does not exist inside that folder.

2a. Let's instead try using `--path` flag: `ng add @ngrx/store --path /projects/whatever-works/src/lib/whatever-works.module.ts`
3a. Error again: `/projects/whatever-works/src/lib/whatever-works.module.ts/app`

The `app` directory seems to be assumed no matter what. Please remove this assumption from schematics and make the schematics work with angular structure.

### Versions of NgRx, Angular, Node, affected browser(s) and operating system(s)

NgRx 14.1.0
Angular 14.2.1

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by timdeschryver** [comment]
Hi, did you try to provide the module option?

```
ng add @ngrx/store --module /projects/whatever-works/src/lib/whatever-works.module.ts
```

**Comment by armodev** [comment]
No, I did not. Will try it next time, now I had to move on with my stuff.

When I read in the docs that I can provide the name of the project to _help locating the module_, I prefer this over providing a long path, prone to typos.
Why isn't it enough to provide name of the project?

Probably there is some reasoning behind this convoluted syntax and redundant-looking flags. It would be good to explain those better so it is clear which takes precedence, what kind of assumptions are behind them, etc.
